spec.md

```diff
@@ -3610,6 +3641,9 @@ type_ref = ( "Probe" | "RWProbe" ) , "<", type , ">" ;
 field = [ "flip" ] , id , ":" , type ;
 type = ( [ "const" ] , ( type_ground | type_aggregate ) ) | type_ref ;
 
+(* Type alias declaration *)
+type_alias_decl = "type", id, "=", type ;
```

Comment:
This should probably be included int he EBNF for `type` as well. This forces questions like can you have a `const` in a typedef, can you apply a `const` onto a named type? Etc.

Reply:
Yeah, that's good point.  I updated EBNF to allow typealias to be quantified by const.